README.md

```diff
@@ -147,18 +159,56 @@ See file `Makefile` which currently supports cross compilation. Just run `make`
 
 Also inspect file `bin/md5Sums.txt` after the cross compilation.
 
-
-## Configuration
+#### API keys
 
 Optionally set the `GOOGLE_API_CLIENT_ID` and `GOOGLE_API_CLIENT_SECRET` environment variables to use your own API keys.
 
-## Usage
+#### Scripts
```

Comment:
I think scripts and automation should be in their own .md file just like
we have platform_packages.md. My reasoning is that automation will
be ran by other folks(yourself and other contributors) and also if
you look in the reasons for why this project was created, we talk
about not having to deal with auto-sync because it requires a lot of
effort and needs more work. I personally don't use any automation
for drive so I'd be more comfortable if you created a separate file and
then linked to it. There also have been other automation script suggestions that have been made in the issues so those folks
can include their scripts in that file when created.